### DISCUSSION TITLE
Is it possible to install the latest version of Postfixadmin with a global install of Composer?

### DISCUSSION THREAD

**Discussion by jnichel** [postfixadmin/postfixadmin#747] [Q&A] [answered]
Hi, forgive me if this is a stupid question, but I'm fairly new to composer.  I like to keep things neat and in a central location, so I have a global install of composer already on the instance that I want to install Postfixadmin on.  Looking through the howto's and the install instruction, I didn't see anything about setting it up this way.  Is it possible?

**Comment by DavidGoodwin** [comment, marked as answer]
Yes - just run `composer install -n` within the Postfixadmin directory.

If you're running Postfixadmin 3.3, I don't think you need to run composer - as it's only really for 'dev' stuff there - see https://github.com/postfixadmin/postfixadmin/blob/postfixadmin_3.3/composer.json (there are no libraries under the 'require' section)

**Reply by jnichel** [reply]
Thank you very much!
